server/bin/www

```diff
@@ -24,6 +25,27 @@ app.set("port", port);
 
 var server = http.createServer(app);
 
+// Socket
+var io = require("socket.io")(server);
+
+io.on("connection", (socket) => {
+  console.log("Connected: ", socket.id);
+
+  let map = new Map();
+
+  socket.on("currentUser", (currentUser) => {
+    map.set(currentUser, socket.id);
+    console.log(map.get(currentUser));
+  });
```

Comment:
You can get the user id from the token in the cookie like this: https://stackoverflow.com/a/33316122, so you don't need the currentUser event